test/EFCore.PG.FunctionalTests/Query/ArrayListQueryTest.cs

```diff
@@ -876,9 +876,9 @@ public override async Task Array_IndexOf2(bool async)
     }
 
     // Note: see NorthwindFunctionsQueryNpgsqlTest.String_Join_non_aggregate for regular use without an array column/parameter
-    public override async Task String_Join_with_array_parameter(bool async)
+    public override async Task String_Join_with_array_of_int_parameter(bool async)
     {
-        await base.String_Join_with_array_parameter(async);
+        await base.String_Join_with_array_of_int_parameter(async);
```

Comment:
This actually seems mis-named (before your PR) - the array of int is a column, not a parameter. Can you please rename to `String_Join_with_array_of_int_column` (also the string variant)? Also,

Reply:
Not sure if something after your
> Also,

was cut off?

Reply:
No, nothing important.